src/System.Management.Automation/engine/remoting/commands/newrunspacecommand.cs

```diff
@@ -1136,6 +1150,26 @@ private List<RemoteRunspace> CreateRunspacesForSSHHostHashParameterSet()
             return remoteRunspaces;
         }
 
+        /// <summary>
+        /// Helper method to create remote runspace based on ProcessId parameter set.
+        /// </summary>
+        /// <returns>Remote runspace that was created.</returns>
+        private List<RemoteRunspace> CreateRunspacesForUseWindowsPowerShellParameterSet()
+        {
+            var remoteRunspaces = new List<RemoteRunspace>();
+
+            NewProcessConnectionInfo connectionInfo = new NewProcessConnectionInfo(this.Credential);
+            connectionInfo.AuthenticationMechanism = this.Authentication;
+            connectionInfo.PSVersion = new Version(5, 1);
+
+            var typeTable = TypeTable.LoadDefaultTypeFiles();
+            string rsName = GetRunspaceName(0, out int rsIdUnused);
+            RemoteRunspace remoteRunspace = (RemoteRunspace)RunspaceFactory.CreateRunspace(connectionInfo, this.Host, typeTable, null, rsName);
```

Comment:
The same for null.

Reply:
Used named arguments like in similar methods for other transports.